ldaptor/__init__.py

```diff
@@ -1,5 +1,5 @@
 """A Pure-Python Twisted library for LDAP"""
-__version__ = "20.0.0rc2.dev0"
+__version__ = "20.0.1.dev0"
```

Comment:
why is this `.dev0`?

Reply:
To distinguish versions in SCM only from versions on PyPI. It's the zest.releaser default

Reply:
You can change the dev_version_template but I don't see much point, it would just be inconsistent with other projects

https://zestreleaser.readthedocs.io/en/latest/entrypoints.html#postrelease-data-dict-items

Reply:
no need to change... we can keep PEP440 convention. is just that I was expecting semver by default